Misc/NEWS.d/next/Library/2022-04-23-03-24-00.gh-issue-91832.TyLi65.rst

```diff
@@ -0,0 +1 @@
+Added 'required' to names list in Lib.argparse.Action.
```

Comment:
Python stdlib doesn’t use a root package name, Lib is just a source directory.
```suggestion
Added 'required' to names list in argparse.Action.
```

Reply:
Same thing.  My point is that it does not matter that a line in the code uses `names`, it is not meaninfgul on its own when read by people in the NEWS.

Reply:
How does this sound then:

Added a parameter "required" to the list returned by `_get_kwargs` in argparse.Action.

Reply:
Ahhh..... Sorry, I'm still new here, and trying to get to know things. Thanks for taking the time to help me understand.

Reply:
No worry!  There is info in the devguide: https://devguide.python.org/pullrequest/

(for future PRs, please don’t force push, it makes reviews harder)